### ISSUE TITLE
Is there a Bert version of the OpenAIGPTLMHeadModel?

### ISSUE THREAD

**Issue by logan-siyao-peng** [huggingface/transformers#12530]
# 🚀 Feature request

I am trying to use the bert-base-chinese model to evaluate the fluency of Chinese texts. I noticed a sample code that uses openai-gpt for English that employes the class OpenAIGPTLMHeadModel. However, since only the bert-base-chinese model is available for Chinese, I am wondering whether there is an equivalence in BERT.

The sample code is attached below:
model = OpenAIGPTLMHeadModel.from_pretrained('openai-gpt',cache_dir='gpt')
tokenizer = OpenAIGPTTokenizer.from_pretrained('openai-gpt',cache_dir='gpt')
tokenize_input = tokenizer.tokenize(sentence)
tensor_input = torch.tensor([tokenizer.convert_tokens_to_ids(tokenize_input)])
loss=model(tensor_input, lm_labels=tensor_input)
print(math.exp(loss))
